### PR TITLE
cloak of darkness and batswarm

### PR DESCRIPTION
**Pull request by ShadowDeath6**
this makes bats actually hostile because I didn't know what to do in order to make them not retaliate against people
it doesn't really do much since bats aren't used at all

also adds cloak of darkness, it got removed when we rebased

### CODE REVIEW

**Comment by Super3222** [comment]
Good content.

File: code/modules/mob/living/simple_animal/hostile/retaliate/bat.dm [hunk 1]
```diff
@@ -35,9 +35,10 @@
 	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
 	minbodytemp = 0
 
-/mob/living/simple_animal/hostile/retaliate/bat/vampire
+/mob/living/simple_animal/hostile/bat/vampire
 	desc = "A rare breed of bat which preys in spaceships. This one seems pretty vampiric."
 	maxHealth = 200
+	gold_core_spawnable = 0
```
Comment (Super3222) [inline code/modules/mob/living/simple_animal/hostile/retaliate/bat.dm:41]: Nice one!

File: code/game/gamemodes/vampire/vampire_powers/vamp_400.dm [hunk 2]
```diff
@@ -35,4 +35,24 @@
...
+		return
+	H.visible_message("<span class='warning'>[H] summons a swarm of bats!</span>")
+	playsound(get_turf(H), 'sound/magic/Ethereal_Enter.ogg', 50, 1, -1)
+	for(var/i = 1, i <= amount_to_spawn, i++)
+		var/mob/living/simple_animal/hostile/bat/B
+		B = new /mob/living/simple_animal/hostile/bat(get_turf(holder.my_atom))
+		B.faction |= "Vampire"
+		if(prob(50))
+			for(var/j = 1, j <= rand(1, 3), j++)
+				step(C, pick(NORTH,SOUTH,EAST,WEST))
```
Comment (ShadowDeath6) [inline code/game/gamemodes/vampire/vampire_powers/vamp_400.dm:58]: This might actually need to be B.step

Comment (Super3222) [reply code/game/gamemodes/vampire/vampire_powers/vamp_400.dm:58]: Shitballs

Comment (Super3222) [reply code/game/gamemodes/vampire/vampire_powers/vamp_400.dm:58]: Also I'm making a new branch for the project. Would you mind pring it to that one when I have it live?

Comment (Super3222) [reply code/game/gamemodes/vampire/vampire_powers/vamp_400.dm:58]: pring this entire pr in general ;)

**Pull request merged**